### DISCUSSION TITLE
Chinese translation file error

### DISCUSSION THREAD

**Discussion by OMEGA-1-HMCL** [RisottoMan/AutoEvent#99] [Q&A]
After copying and replacing the translation file, I encountered subtitle errors while loading the mini game

**Comment by RisottoMan** [comment]
Fixed

**Comment by kldhsh123** [comment]
尝试使用最新翻译，之前的翻译版本落后了。
还有问题请携带图片询问。
Try to use the latest translation, the previous translation is lagging behind.
If you have any questions, please bring pictures to ask.
